k8s/perftest/cluster-00/static-overlay/static/fluxcloud/fluxcloud.yaml

```
@@ -0,0 +1,11 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: fluxcloud
+  namespace: admin
+spec:
+  values:
+    environment:
+      SLACK_CHANNEL: "#aks-monitor-test"
```

perftest? or is it too long?
```suggestion
      SLACK_CHANNEL: "#aks-monitor-perftest"
```